### PR TITLE
to_prod list update July 3th 

### PR DESCRIPTION
**Pull request by vv-monsalve**
This quick update to the list includes the remaining Playwrite projects plus a couple of new additions.

### CODE REVIEW

**Review by chrissimpkins — COMMENTED** [review]
@vv-monsalve I haven't had a chance to review this PR yet.  Took a quick look through changes and I think that we need to block the COLRv1 Kalnia Glaze in Workspace before the prod push.   Mind removing that one from the list?

cc @garretrieger

**Comment by chrissimpkins** [comment]
@m4rc1e we've had CI fails on every list PR over the last few weeks.  If this is a test that we are ignoring can we remove it altogether?

**Comment by vv-monsalve** [comment]
@chrissimpkins I merged it since the last time we did a quick update to the lists, you said there was no need for you to review it if there were no projects to review in WS before merging. 

**Kalnia Glaze**
I added Kalnia Glaze after checking about the `SVG` table with Garret, where he said:

> Ah ok, in that case I think we're good to remove Kalnia Glaze from the sandbox denylist.

**CI Fail**
Before merging this one, I checked that the latest PRs had been merged with that CI Fail. However, I opened a [PR to fix the broken link in one of the Knowledge pages](https://github.com/google/fonts/pull/7918). The last time I was on call, Dave fixed the source of the problem reported by the Knowledge CI by updating the broken link reported and said that we could continue doing so when seeing these cases.

File: to_production.txt [hunk 1]
```diff
@@ -1,11 +1,26 @@
 # New
+ofl/eduauvicwanthand # https://github.com/google/fonts/pull/7628
+ofl/kalniaglaze # https://github.com/google/fonts/pull/7864
```
[the remark this answers is not in the document]
Comment (vv-monsalve) [reply to_production.txt:3]: Solved in #7922

**Pull request merged**